89.md

```diff
@@ -0,0 +1,53 @@
+NIP-89
+======
+
+Bitcoin Transaction Broadcasting Events
+-------------------------
+
+`draft` `optional` `author:benthecarman`
+
+A special event with kind `28333`, that contains an encoded bitcoin transaction.
+The bitcoin transaction is encoded as a base64 string, and is contained in the `content` field.
+
+The network magic is contained in the `tags` field. This is to identify the network that the transaction is for.
+The common ones are:
+ - mainnet: f9beb4d9
+ - testnet: 0b110907
+ - signet (default): 0a03cf40
+ - regtest: fabfb5da
+
+If another value is present it is likely a custom signet.
+
+This is an ephemeral event, and is not stored on the server. So relays or other custom clients could receive these
+events, and broadcast them to the bitcoin network.
+
+For example:
+
+```json
+{
+  "kind": 28333,
+  "content": "AgAAAAABAtkXo8t2LfqIwQ+BPQg/LZdqAtAb61bY0zIjFvvxMTtGAAAAAAD9////j8wvOaFL5Dz+0CTKXNKxxv1GrusMps3aodsXQGp95xgAAAAAAP3///8CLAPTBQAAAAAiUSBIKoy1/JK1GdfWhii8KLfBooZg3qT7ImuwwwC6t2eT6oiKAQAAAAAAIlEgZtFI4Bx2rDqBe4X9+lzEhh7Kxca3tNE9l/QfH/ZghQECRzBEAiBV2O5+jskV8/H+UuzgRuku2gCAwlmErieXrS5scbongAIgLWD6uN8BGpyE7gNtX4jxnkNShY8yjhdcjhsHR4RAWJEBIQLlCnhSR+K9YoAngUWj94mptHhcVYJGi+rT3lwApIekkQJHMEQCIDwrK0YHftZ2BQsBjHdJb89Tlpzvc3z8E1wGgbqIVRBQAiB6J0UXuPjib+geVmaZeoq4vYD+fDz835o5OJRg9ITFDAEhA1oq2Zj0d4XbATjc3OUSXSJDvvIosDeAsHyegwk4fkcTNAslAA==",
+  "tags": [
+    ["magic", "f9beb4d9"]
+  ],
+  ...other fields
+}
+```
+
+## Uses
+
+This offers a way for users to broadcast bitcoin transactions in a private way without having to connect to the bitcoin
+network directly.
+
+Today, users have to connect to the bitcoin network directly or use an api like mempool.space to broadcast transactions.
+This is not private, as you can leak your ip address to the service or a bitcoin node.
+
+Nostr provides a better opportunity for users to broadcast transactions in a private way. Nostr relays are at most
+bitcoin adjacent and have a much lower risk of being run by a chain analysis company, thus providing a possibly better
+way to broadcast transactions.
+
+## Recommendations
+
+Clients should generate an ephemeral keypair for each transaction they want to broadcast. This keypair should only ever
+be used for broadcasting that transaction. This is to best preserve the privacy of the user and to not link any extra
+metadata to the transaction.
```

Comment:
Not sure if it's worth mentioning, but less is really more when it comes to broadcasting the transaction, especially if it's for privacy reasons.

If someone is doing this for privacy reasons, broadcasting to a lot of relays will maximize the chance to broadcast to a relay that is monitoring IPs and associating events with them. For now, it might not be the case that CA companies are running relays just to eavesdrop on bitcoin transactions being broadcasted, but over time there could be a bunch of events that are juicy from an bitcoin analysis perspective.

Reply:
Added a section about it